.github/workflows/test_nightly_macos_apple_silicon.yml

```diff
@@ -1,7 +1,7 @@
 on:
   workflow_dispatch:
-  schedule:
-    - cron:  '0 13 * * *'
+  release:
+    types: [edited]
```

Comment:
This can be run as soon as the nightly is uploaded, no need to wait until a specific time.